.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+#### Motivation
```

Comment:
I'm currently experimenting with the format. Markdown looks nice in UIs that support it and helps visually separate different sections/titles and still looks reasonable as plain text.